build-system/babel-config/import-resolver.js

```diff
@@ -18,11 +18,11 @@
 const fs = require('fs');
 const path = require('path');
 
-const JSCONFIG_PATH = path.join(__dirname, '..', '..', 'jsconfig.json');
+const JSCONFIG_PATH = path.join(__dirname, '..', '..', 'tsconfig.json');
```

Comment:
Nit: Since this switched to tsconfig, might update the variable name here.

Reply:
Done